_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -183,7 +183,23 @@ And on Debian/Ubuntu installations with Passenger, run:
 
     service apache2 restart
 
-#### Optional Step 5 - Cleanup of RPMs and repositories
+#### Optional Step 5 - Reclaim database space
+
+After database migrations, some space can sometimes be reclaimed. It's a good idea to perform a *full* database vacuum for Postgres rather than relying on the autovacuum feature to claim maximum space possible. For PostgreSQL do the following:
```

Comment:
It's called PostgreSQL, not Postgres :)

Reply:
"for postgresql…" should probably be an own paragraph.

Reply:
Well it is accepted alias, it was a main name candidate for some time actually but the idea got dropped eventually: https://wiki.postgresql.org/wiki/ProjectName

But I will use the official name, not looking good to mix them in a single doc.